Altis_Life.Altis/init.sqf

```diff
@@ -7,7 +7,9 @@
 */
 StartProgress = false;
 
-[] execVM "briefing.sqf"; //Load Briefing
+if (hasInterface) then {
```

Comment:
Some peoples place more than this in their inits. Would it not be better to use this:

`if (!hasInterface) exitWith {
	StartProgress = true;
};`